drafts/25-xxx-specifications.txt

```diff
@@ -413,22 +413,141 @@ in28. Unlike INCLUDE lines (see the section on "INCLUDE lines"),
       the #include directive is not prohibited from including the
       same source file at a deeper level of nesting.
 
+
 3.5 The '#if', '#ifdef', '#ifndef', '#elif', '#elifdef',
-      '#elifndef', '#else', '#endif' directives
+      '#elifndef', '#else', '#endif' conditional directives
 ---------------------------------------------------------
 
-Example syntax:
+Example syntax (extra spacing for illustration purposes only):
+    General form:
+        <if-head>  <lines>                        #endif EOL
+        <if-head>  <lines>  <if-elif>             #endif EOL
+        <if-head>  <lines>  <if-elif>  <if-else>  #endif EOL
```

Comment:
Duplicate `<lines>` in `<if-head>`
```suggestion
        <if-head>                        #endif EOL
        <if-head>  <if-elif>             #endif EOL
        <if-head>  <if-elif>  <if-else>  #endif EOL
```

Reply:
Thanks. I missed this when refactoring the grammar on the fly.